frontend_app_html-document/src/container/PopupCreateHtmlDocument.jsx

```diff
@@ -79,7 +80,7 @@ class PopupCreateHtmlDocument extends React.Component {
   handleChangeNewContentName = e => this.setState({newContentName: e.target.value})
 
   handleClose = () => GLOBAL_dispatchEvent({
-    type: 'hide_popupCreateContent', // handled by tracim_front:dist/index.html
+    type: CUSTOM_EVENT.HIDE_POPUP_CREATE_CONTENT, // handled by tracim_front:dist/index.html
```

Comment:
now we have a variable, it is easier to find where are the usages of a given custom event.
So you can remove this kind of comments